src/listwiz/sorting.py

```diff
@@ -21,7 +21,7 @@ def merge_sort(l):
     sorted_list
     """
     # If there is a single item, the list is already sorted, return.
-    if len(l) == 1:
+    if 0 <= len(l) <= 1:
```

Comment:
```suggestion
    if len(l) <= 1:
```
I think it would be clearer to delete this, since the length cannot be negative.

(since it is a small change, one can just apply it.  Often I would just do that as a maintainer)